Bluemix deployment failing

Atm, bluemix deployment is failing with the following error:
`Cloning the 'master' branch from repo 'https://P90MSG01:3041/git/treba/loklak-server-treba-1522'
Buildfile: /home/pipeline/03f83139-311d-4f15-a7d8-81b38f53a0b2/build.xml

init:

build:
    [mkdir] Created dir: /home/pipeline/03f83139-311d-4f15-a7d8-81b38f53a0b2/classes
     [echo] loklak: /home/pipeline/03f83139-311d-4f15-a7d8-81b38f53a0b2/build.xml
    [javac] Compiling 149 source files to /home/pipeline/03f83139-311d-4f15-a7d8-81b38f53a0b2/classes
    [javac] javac: invalid target release: 1.8
    [javac] Usage: javac <options> <source files>
    [javac] use -help for a list of possible options

BUILD FAILED
/home/pipeline/03f83139-311d-4f15-a7d8-81b38f53a0b2/build.xml:89: Compile failed; see the compiler error output for details.

Total time: 0 seconds

Finished: FAILED`

That happened to @sudheesh001 and me (tested with master branch)

Seems to be some issue with java 8

--- a/.bluemix/pipeline.yml
+++ b/.bluemix/pipeline.yml
@@ -13,6 +13,7 @@ stages:
     build_type: ant
     script: |-
       #!/bin/bash
+      export JAVA_HOME=/opt/IBM/java8
       ant
 - name: Deploy Stage
   inputs:

